cpplint/cpplint.py

```diff
@@ -5399,9 +5410,7 @@ def ExpectingFunctionArgs(clean_lines, linenum):
     )
 
 _HEADERS_MAYBE_TEMPLATES = (
-    ('<algorithm>', ('copy', 'max', 'min', 'min_element', 'sort',
-                     'transform',
-                    )),
+    ('<algorithm>', ('copy', 'max', 'min', 'min_element', 'sort',)),
```

Comment:
we should have std:: for these

Reply:
Very good point. I'll add std:: if works correctly.